test/test_queries.go

```diff
@@ -16,12 +16,13 @@ func testClientPair(t *testing.T, src, dst *Chain) {
 
 // testClient queries clients and client for dst on src and returns a variety of errors
 // testClient expects just one client on src, that for dst
-// TODO: we should be able to find the chain id of dst on src, add a case for this in each switch
+// a localhost client is always created upon init genesis
 func testClient(t *testing.T, src, dst *Chain) {
+	// require 2 clients on src, localhost and dst
 	clients, err := src.QueryClients(1, 1000)
 	require.NoError(t, err)
-	require.Equal(t, len(clients), 2)
-	require.Equal(t, clients[0].GetID(), src.PathEnd.ClientID)
+	require.Equal(t, 2, len(clients))
```

Comment:
hmm I just noticed that the other chains use this func as well. This will fail for any chain that doesn't init a localhost. Perhaps it is best to just remove this query all together and just do the direct query for the client as done below